### PR TITLE
create the ECR Docker registry for uat-claim-criminal-injuries-compen…

### PR DESCRIPTION
**Pull request by ermiasworku**
…sation

### CODE REVIEW

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/uat-claim-criminal-injuries-compensation/resources/ecr.tf [hunk 1]
```diff
@@ -0,0 +1,28 @@
+/*
+ * Make sure that you use the latest version of the module by changing the
+ * `ref=` value in the `source` attribute to the latest version listed on the
+ * releases page of this repository.
+ *
+ */
+module "cica_ecr_credentials" {
+  source     = "github.com/ministryofjustice/cloud-platform-terraform-ecr-credentials?ref=3.1"
```
Comment (digitalronin) [inline namespaces/live-1.cloud-platform.service.justice.gov.uk/uat-claim-criminal-injuries-compensation/resources/ecr.tf]: The latest version of the ECR credentials module is 3.2 - please use that as the 'ref' value here.

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/uat-claim-criminal-injuries-compensation/resources/ecr.tf [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  repo_name  = "cica-repo-uat"
+  team_name  = "cica"
+  aws_region = "${var.aws_region}"
```
Comment (digitalronin) [inline namespaces/live-1.cloud-platform.service.justice.gov.uk/uat-claim-criminal-injuries-compensation/resources/ecr.tf]: Please replace `aws_region` with:
```
  providers = { 
    aws = "aws.london"
  }
```
(see [Usage](https://github.com/ministryofjustice/cloud-platform-terraform-ecr-credentials#usage))

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/uat-claim-criminal-injuries-compensation/resources/main.tf [hunk 3]
```diff
@@ -0,0 +1,3 @@
+terraform {
```
Comment (digitalronin) [inline namespaces/live-1.cloud-platform.service.justice.gov.uk/uat-claim-criminal-injuries-compensation/resources/main.tf]: This should be a copy of this file:

https://github.com/ministryofjustice/cloud-platform-terraform-ecr-credentials/blob/master/examples/main.tf

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/uat-claim-criminal-injuries-compensation/resources/variable.tf [hunk 4]
```diff
@@ -0,0 +1,39 @@
+variable "aws_region" {
```
Comment (digitalronin) [inline namespaces/live-1.cloud-platform.service.justice.gov.uk/uat-claim-criminal-injuries-compensation/resources/variable.tf]: Lines 1-12 should be deleted - those values will come from main.tf

**Comment by ermiasworku** [comment]
@digitalronin Do you still want me to update ecr.tf and main.tf config? Or Shall I merge it?

**Comment by ermiasworku** [comment]
@digitalronin As requested I have updated main.tf ,variable.tf and ecr.tf

**Comment by mtrblmoj** [comment]
Closing this. Starting afresh elsewhere.

**Pull request closed without merging**